.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,22 +1,52 @@
 ---
-name: Bug Report
-about: Reporting issues with the VA API Platform
-
+name: 🔴 Bug Report
+about: For reporting issues with the VA API Platform
 ---
 
-## APIs Affected
+⚠️ Please read before submitting: As this is a public forum, please do not include your VA API OAuth Client ID, VA API OAuth Client Secret, VA API key, or any Personally Identifiable Information (PII) or Protected Health Information (PHI) in your report. For test apps, please make sure you are using your registered redirect URL, not the URL provided in the example.
+
+
+## Email address
+
+<!-- Please include your email address. -->
+
+
+## APIs affected
+
+<!-- Please list affected APIs (if applicable). -->
+
 
-<!-- Please list affected APIs -->
+## What I expected to happen
 
-## Brief Summary 
+<!-- Please provide a step by step summary of what the expected behavior was. -->
+1. 
+2. 
+3. 
 
-<!-- One or two sentence summary of the issue. --> 
 
-## Severity of the Issue
+## What actually happened
+
+<!-- Describe in detail what went wrong. Screenshots, gifs, and videos are encouraged. -->
+
+
+## Severity of the issue
```

Comment:
Given that we'll most likely have multiple versions of the same API supported at some point, should we have a version pulldown for the bug (and possibly documentation) templates?

Reply:
@davem-va I can add a field for this, for sure. It has to be free text though as these templates just suggest a starting point as markdown when users are creating an issue.

Reply:
Fixed in [bf313c3](https://github.com/department-of-veterans-affairs/vets-api-clients/pull/41/commits/bf313c36b0d1e6b28d444d7f83f720fbd41f7681).